packages/cli/src/active-workflow-manager.ts

```diff
@@ -532,6 +531,13 @@ export class ActiveWorkflowManager {
 				});
 			}
 
+			if (['init', 'leadershipChange'].includes(activationMode) && !dbWorkflow.active) {
```

Comment:
Do we need the `['init', 'leadershipChange'].includes(activationMode)` check? Wouldn't we want to skip the activation in other cases as well? Would also simplify things if we have one less conditional

Reply:
This method is also used elsewhere, e.g. by `/activate` with mode `activate` for a workflow that by definition is currently inactive, so we need to tighten here.

Reply:
Oh right good point, it hasn't been marked as active yet